spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/eureka/ZoneUtils.java

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2013-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.netflix.ribbon.eureka;
+
+import org.springframework.util.StringUtils;
+
+/**
+ * Utility class for dealing with zones.
+ * @author Ryan Baxter
+ *
+ */
+public class ZoneUtils {
+
+	/**
+	 * Approximates Eureka zones from a host name. This method approximates the zone to be
+	 * everything after the first "." in the host name.
+	 * @param host The host name to extract the host name from
+	 * @return The approximate zone
+	 */
+	public static String extractApproximateZone(String host) {
```

Comment:
Other ways to approximate the zone from the host name?  Right now there is only a property to enable this functionality based off the host name.  If you think there might be more than one way of doing that than I think an interface might make sense.